### PR TITLE
Linux wiki

### PR DESCRIPTION
**Pull request by coymeetsworld**
Rewrote wiki on Linux to include more introductory topics including Origins of Linux, Linux usage (including its role in web hosting), Why use Linux and how to obtain/setup Linux.

### CODE REVIEW

File: Linux.md [hunk 1]
```diff
@@ -2,36 +2,80 @@
...
...//en.wikipedia.org/wiki/Free_software_movement)'s ideals (Individuals who use and collaboratively contribute to open and free software). It was also responsible for starting the [GNU Project](https://en.wikipedia.org/wiki/GNU_Project) in 1984 which was the first attempt at a free OS that was not UNIX (GNU is a recursive algorithm for 'GNUs not UNIX'). The GNU Project would include everything useful found on a UNIX system but written from scratch. More importantly, along with the OS the source code would also be legally available to those who wanted it. The idea was that freely exchanging the source code and inner knowledge of the OS would encourage collaborative development, which in turn would create a better OS product. The GNU Project eventually led to the [GNU Public License](https://en.wikipedia.org/wiki/GNU_General_Public_License) (GPL) which legalized all software released under it to be made freely available as well as require providing the source code. While the GPL was a success and still used today to license free software, the GNU Project never took off as planned. Another hacker stepped in and provided his implementation of a free Operating System to the world instead.
```
Comment (koustuvsinha) [inline Linux.md]: recursive ~~algorithm~~ acronym, also add a [reference](https://en.wikipedia.org/wiki/Recursive_acronym)

File: Linux.md [hunk 2]
```diff
@@ -2,36 +2,80 @@
...
 
-    Only manages CPU, Memory, IPC (Inter Process communication), rest is considered as an accessory and can be handled in user mode.
 
-- **Monolithic Kernel:**
+[Linux Torvalds](https://en.wikipedia.org/wiki/Linus_Torvalds), a Finnish student from the University of Helsinki, first created [Linux](https://en.wikipedia.org/wiki/Linux) while experimenting with a free and smaller version of UNIX used for educational purposes called [MINIX](https://en.wikipedia.org/wiki/MINIX) which was developed by [Andrew Tantebatum](https://en.wikipedia.org/wiki/Andrew_S._Tanenbaum). Linus wanted to have a similar free OS that didn't have the constraints of being limited to pure educational use. He created Linux which he licensed under the GPL and distributed on the Internet in 1991. Linus didn't use any code from MINIX, but he incorporated the features found in MINIX/UNIX which is why Linux is often called ["UNIX-like"](https://en.wikipedia.org/wiki/Unix-like) or a "UNIX clone".
```
Comment (koustuvsinha) [inline Linux.md]: typo : Linus Torvals

**Comment by alayek** [comment]
@theflametrooper @SeanFCC you guys might want to take a look. You have an eye for grammar and sentence construction!

File: Linux.md [hunk 3]
```diff
@@ -2,36 +2,80 @@
...
+- **Interoperability with other Operating Systems.**
+  Linux has driver support for NTFS and HFS+ filesystems (used by Windows and Macs) and also [Samba](https://www.samba.org/) for file/print service support on Windows machines. 
+- **Better hardware support.**
+  For other OSs, usually a user would have to go to the manufacturer's website to get driver support for different types of hardware. The Linux kernel supports a majority of hardware automatically via plug-and-play (largely in part because of the open source community).
+- **Software Availability/Compatibility.**
+    Linux has a rich set of applications that are available. It has [office applications](https://www.libreoffice.org/discover/libreoffice/), [browsers](https://www.google.com/chrome/browser/desktop/), [media players](http://www.videolan.org/vlc/), [image](https://www.gimp.org/)/[video processing](http://www.openshot.org/), etc. For video gaming, [Steam](http://store.steampowered.com/about/) runs on Linux which has many supported games. Windows applications can also run within Linux (see [Wine](https://www.winehq.org/) for details).
```
Comment (koustuvsinha) [inline Linux.md]: in browsers provide Chrome **&** Mozilla. would be disrespectful not to mention Mozilla in a Linux about me :stuck_out_tongue:

File: Linux.md [hunk 4]
```diff
@@ -2,36 +2,80 @@
...
+- **It's Free.**
+  Hard to beat that!
 
-> Linux is simply everywhere.
+[Back to top](#table-of-contents)
+
+## How to get Linux
+
+While most vendors provide one OS that gets incremental updates and new versions over time (e.g. Windows, Mac OSX, Solaris), Linux has hundreds of different versions to choose from created by different vendors. This is in part because Linux is freely-distributed code which can be tailored to specific user's needs. While the [Linux Kernel](https://en.wikipedia.org/wiki/Linux_kernel) typically stays the same (core functionality of the OS handling tasks like network management, disk usage, memory utilization, I/O etc.), the many different versions of Linux differ in parts such as which GUI it uses (i.e. [Gnome](https://en.wikipedia.org/wiki/Gnome), [KDE](https://en.wikipedia.org/wiki/KDE), [XFCE](https://en.wikipedia.org/wiki/Xfce), or none at all), system tools, environments, package managers, and applications provided. This unique collection of software along with the kernel define what we call a [Linux Distribution](https://en.wikipedia.org/wiki/Linux_distribution) (distro for short).
```
Comment (koustuvsinha) [inline Linux.md]: would be great if you can link the [Linux kernel repository](https://github.com/torvalds/linux) here. Also link the official webpages for [Gnome](https://www.gnome.org/), [KDE](https://www.kde.org/), and [XFCE](http://www.xfce.org/)

**Comment by koustuvsinha** [comment]
quite wonderfully written @coymeetsworld ! fix those little typos and links and would be perfect :smile:

**Comment by Rafase282** [comment]
Please make sure your fork is up to date, this PR includes unrelated commits.

**Comment by coymeetsworld** [comment]
Sure, thanks for the input @koustuvsinha @Rafase282. I will take a look and attempt to correct those issues.

**Comment by coymeetsworld** [comment]
Made changes, but I will submit a new PR so the unrelated change is not included.

**Pull request closed without merging**